Gemfile.lock

```diff
@@ -202,6 +204,3 @@ DEPENDENCIES
   unicorn
   will_paginate_mongoid (~> 2.0)
   yajl-ruby
-
-BUNDLED WITH
```

Comment:
Thanks for the details.

On Tue, Jul 5, 2016 at 5:24 PM, Brian Beggs <notifications@github.com>
wrote:

> In Gemfile.lock
> <https://github.com/edx/cs_comments_service/pull/185#discussion_r69640880>
> :
>
> > @@ -202,6 +204,3 @@ DEPENDENCIES
> >    unicorn
> >    will_paginate_mongoid (~> 2.0)
> >    yajl-ruby
> > -
> > -BUNDLED WITH
>
> I will remove it, this has prevented us from deploying the app in the past
> due to bundle version differences.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/edx/cs_comments_service/pull/185/files/3c2ea074f800998be1f39520e353438228bd7bce#r69640880>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AN5rPccVkTOOYrQSfdBJEzBJDJMlVzusks5qSsuqgaJpZM4JFhMu>
> .
>



-- 

*Robert Raposa*

edX | Software Architect  | rraposa@edx.org

141 Portland Street, 9th floor

Cambridge, MA 02139
http://www.edx.org <http://www.edxonline.org/>

[image:
http://www.e-learn.nl/media/blogs/e-learn/edX_Logo_Col_RGB_FINAL.jpg?mtime=1336074566]